### PR TITLE
Image&input

### PR DESCRIPTION
**Pull request by sahiba0915**
Added image and input component

### CODE REVIEW

File: component/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.avatar-title {
+  font-size: 2rem;
+  padding-top: 1.5rem;
+  color: black;
+}
+.avatar {
+  display: flex;
+  width: 100px;
```
Comment (akash-1318) [inline component/avatar/avatar.css:29]: use rem here

File: component/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.badge-btn {
+  position: relative;
+  display: flex;
+  display: inline;
+  align-items: center;
+  justify-content: center;
+  width: 100px;
+  height: 100px;
```
Comment (akash-1318) [inline component/badge/badge.css:41]: use rem here.

**Review by akash-1318 — COMMENTED** [review]
Looks clean with proper formatting from my end it's ready to merge.

File: component/alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      <i class="nav-icon fab fa-uikit"></i>
+      <div class="nav-brand">UiPlugIn</div>
+      <ul class="list-non-bullet nav-pills">
+        <li class="list-item-inline">
+          <a class="link" href="/" target="_blank">Home</a>
+        </li>
+
+        <li class="list-item-inline">
+          <a class="link" href="/docs.html" target="_blank">Documentation</a>
+        </li>
```
Comment (darshandamre) [inline component/alert/alert.html:32]: you should not use target blank if the link goes to some another page of the same website, same thing in rest of the pages

Comment (sahiba0915) [reply component/alert/alert.html:32]: done

File: component/badge/badge.css (hunk 2 above)
Comment (sahiba0915) [reply component/badge/badge.css:41]: done

File: component/avatar/avatar.css (hunk 1 above)
Comment (sahiba0915) [reply component/avatar/avatar.css:29]: done

**Comment by sahiba0915** [comment]
Merging Image and Input component

**Pull request merged**